lib/Constants/flags.ts

```diff
@@ -3,6 +3,10 @@ export enum Storage {
   RANDOMS = 'randoms', // used to store no-duplicate random node choices
 }
 
+export enum TurnType {
+  STOP_TYPES = 'stopTypes',
```

Comment:
do we need to include the `stopAll` logic from `general-runtime` as well? 

We might want to move all the _v1  handler over here then?

Reply:
idk, `_v1` handler for general is very different from the one in alexa/google. Plus all the `event matching` logic inside the handler is platform specific (and uses helper functions specific to the repo) and that was the reason we created `_v1` in the each runtime in the first place